Libplanet/Assets/Currency.cs

```diff
@@ -530,7 +530,7 @@ private Currency(SerializationInfo info, StreamingContext context)
         /// </returns>
         /// <exception cref="ArgumentException">Thrown when the given <paramref name="ticker"/>
         /// is an empty string.</exception>
-        [Obsolete]
+        [Obsolete("Obsolete! Use Currency.Capped() or Currency.Uncapped() for new currencies.")]
         public static Currency LegacyUntracked(
```

Comment:
I think `Legacy()` might be enough since there is no `LegacyTracked()`? 😶

Reply:
I had a forethought that there might be another instance where we might make another legacy currency, but again it might have been unnecessary... 🤔